Say Their Names/Source/Controller/Petitions/PetitionsUI.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  PetitionsUI.swift
+//  Say Their Names
+//
+//  Created by Kyle Lee on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Petitions
+final class PetitionsUI: UICollectionView {
```

Comment:
To have some consistency with the type used, it will make more sense to name it as `PetitionCollectionView` instead of `PetitionsUI`, after all, this is only a CollectionView and does not reflect the whole content of the whole screen. For the viewCode of the whole screen, we should have something like `PetitionViewCode` way.

Reply:
As of right now, the designs call for only a collection view, but this may change over time. Maybe it would be best for me to change this to a view that has a collection view subview